### PR TITLE
Add confirmation dialogs and success pages to government review

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

**Review by trslater — APPROVED** [review]

File: portal-frontend/src/app/features/applications/review-submission/success/success.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      ></span
+    ><br /><br /><br />
+    {{ localGovernmentName }} will be cc'd on the email notifying the applicant that the application has been forwarded
+    to the ALC.<br /><br />ALC Staff may contact you with application related inquiries prior to the Commission review.
+  </p>
+</ng-container>
+<ng-container *ngIf="!didAuthorize">
+  <h2 class="heading">Application ID: {{ fileId }} has been refused by the {{ localGovernmentName }}.</h2>
+  <p>
+    <span class="subheading1"><strong>Your review and decision has been submitted to the ALC.</strong></span
```
Comment (sandratoh) [inline portal-frontend/src/app/features/applications/review-submission/success/success.component.html:16]:
```suggestion
    <span class="subheading1"><strong>Your review and decision have been submitted to the ALC.</strong></span
```

File: portal-frontend/src/app/features/applications/review-submission/review-submit/review-submit.component.ts [hunk 2]
```diff
@@ -102,8 +109,23 @@ export class ReviewSubmitComponent implements OnInit, OnDestroy {
   async onSubmit() {
     const isValid = this.runValidation();
     if (isValid && this.fileId) {
-      await this.applicationReviewService.complete(this.fileId);
-      await this.router.navigateByUrl(`/application/${this.fileId}`);
+      const government = await this.loadGovernment(this.localGovernmentUuid);
+      const governmentName = government?.name ?? 'selected local / first nation government';
```
Comment (sandratoh) [inline portal-frontend/src/app/features/applications/review-submission/review-submit/review-submit.component.ts:113]:
```suggestion
      const governmentName = government?.name ?? 'selected local / First Nation government';
```

File: portal-frontend/src/app/features/applications/review-submission/review-submit-fng/review-submit-fng.component.ts [hunk 3]
```diff
@@ -89,8 +96,23 @@ export class ReviewSubmitFngComponent implements OnInit, OnDestroy {
   async onSubmit() {
     const isValid = this.runValidation();
     if (isValid && this.fileId) {
-      await this.applicationReviewService.complete(this.fileId);
-      await this.router.navigateByUrl(`/application/${this.fileId}`);
+      const government = await this.loadGovernment(this.localGovernmentUuid);
+      const governmentName = government?.name ?? 'selected local / first nation government';
```
Comment (sandratoh) [inline portal-frontend/src/app/features/applications/review-submission/review-submit-fng/review-submit-fng.component.ts:100]:
```suggestion
      const governmentName = government?.name ?? 'selected local / First Nation government';
```

**Review by sandratoh — APPROVED** [review]
Minor typo suggestions but LGTM 👍

**Pull request merged**